tests/data/dataset_readers/stanford_sentiment_tree_bank_test.py

```diff
@@ -0,0 +1,94 @@
+# pylint: disable=no-self-use,invalid-name
+import pytest
+
+from allennlp.data.dataset_readers import StanfordSentimentTreeBankTokensDatasetReader
+from allennlp.common.util import ensure_list
+
+class TestStanfordSentimentTreebankReader():
+    @pytest.mark.parametrize("lazy", (True, False))
+    def test_read_from_file(self, lazy):
+        reader = StanfordSentimentTreeBankTokensDatasetReader(lazy=lazy)
+        instances = reader.read('tests/fixtures/data/sst.txt')
+        instances = ensure_list(instances)
+
+        instance1 = {"tokens": ["The", "actors", "are", "fantastic", "."],
+                     "label": "4"}
+        instance2 = {"tokens": ["It", "was", "terrible", "."],
+                     "label": "0"}
+        instance3 = {"tokens": ["Chomp", "chomp", "!"],
+                     "label": "2"}
+
+        assert len(instances) == 3
+        fields = instances[0].fields
+        assert [t.text for t in fields["tokens"].tokens] == instance1["tokens"]
+        assert fields["label"].label == instance1["label"]
+        fields = instances[1].fields
+        assert [t.text for t in fields["tokens"].tokens] == instance2["tokens"]
+        assert fields["label"].label == instance2["label"]
+        fields = instances[2].fields
+        assert [t.text for t in fields["tokens"].tokens] == instance3["tokens"]
+        assert fields["label"].label == instance3["label"]
+
+    def test_use_subtrees(self):
+        reader = StanfordSentimentTreeBankTokensDatasetReader(use_subtrees=True)
```

Comment:
Can you build this object `from_params` in one of these tests, so that that block of code gets tested, too?  Or just add a test that explicitly checks that parameters are set, but doing it in line for one of these variants seems fine to me, too.